resources/src/idea/JavaActions.xml

```diff
@@ -127,7 +124,7 @@
       <add-to-group group-id="XDebugger.Frames.Tree.Popup"/>
       <add-to-group group-id="XDebugger.Actions"/>
     </action>
-    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" text="Get thread dump" icon="AllIcons.Actions.Dump">
+    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" icon="AllIcons.Actions.Dump">
```

Comment:
Hi,

I'd like to see every string of IntelliJ available for translation in resource bundles as it's important for our project; as so, I (we) sometimes identify these kind of things.
However we're not yet using the most recent code so there's no possibility of contributing our changes directly; but, as we explore the new code, we can start to contribute (but as my last PR shown, we need to know the code better (and how it behaves) so that everything is compact and no duplication of code/functionality exists).
For the DumpThreads action, it's something I consider kind of annoying on IntelliJ: when I hover a menu entry, an icon or anything else that will trigger an action, I like to see the description on the Status Bar, however there're several that don't have description and what is shown it the last message sent to the Status Bar.
You can see this behaviour in the following pic:
![withandwithoutdescription](https://cloud.githubusercontent.com/assets/6816863/25445471/6d859e82-2aa6-11e7-90bc-cc7e81f69945.png)

As for being that specific action was pure coincidence: I stumbled upon it and made the change :-)